### PR TITLE
adding spotbugs checks

### PR DESCRIPTION
**Pull request by itsAftabAlam**
adding spotbugs checks for more deeper checks

### Submitter checklist
- [x] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [x] Ensure that the pull request title represents the desired changelog entry
- [x] Please describe what you did

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -30,6 +30,8 @@
 
     <properties>
         <jenkins.version>2.401.3</jenkins.version>
+        <spotbugs.effort>Max</spotbugs.effort>
```
Comment (damianszczepanik) [inline pom.xml:33]: Resolve conflict and provide some doc reference

**Review by damianszczepanik — CHANGES_REQUESTED** [review]

**Comment by itsAftabAlam** [comment]
I hope this resolves the issue

File: pom.xml (hunk 1 above)
Comment (itsAftabAlam) [reply pom.xml:34]: I hope this resolves the issue

Comment (jonesbusy) [reply pom.xml:34]: https://www.jenkins.io/doc/developer/tutorial-improve/add-more-spotbugs-checks/#increase-spotbugs-checks

**Pull request merged**